src/pages/instances/InstanceSnapshots.tsx

```diff
@@ -172,7 +193,10 @@ const InstanceSnapshots: FC<Props> = ({ instance }) => {
                   aria-label="Search for snapshots"
                 />
               </div>
-              <ConfigureSnapshotsBtn instance={instance} />
+              <ConfigureSnapshotsBtn
+                instance={instance}
+                className="u-no-margin--right"
```

Comment:
I think we can keep this className inside the component, it won't do any harm in the empty state usage.

Reply:
It was needed to move it out, and it still will be even after the change suggested above, in order to avoid the two buttons to be "glued together" with no margin between them. And in general, a reusable component like this one should not "force" a style (like the no-margin--right) that applies to one use case only.